gtk/src/light/gtk-3.0

```diff
@@ -0,0 +1 @@
+gtk-3.20
```

Comment:
Hm. Symlinking this seems somewhat suspicious to me. I think the gtk-3.0 theme is meant to support things compiled with gtk < 3.20. (Which I guess may be relevant for some proprietary software? Don't know how much uses gtk.) Does making this a symlink break that compatibility? Are other themes/distros using a symlink like this?

Reply:
If this is only an issue for supporting older distros, then I guess that's not a problem.

Reply:
We've had the 3.0 version symlinked to the latest 3.x version before (see #168 and #169 ). As an alternative, Adwaita does not include any 3.20 folder and instead relies exclusively on the 3.0 folder, meaning the contents of the 3.0 folder are used regardless of version.

Reply:
Ah, I hadn't compared the packaged version of Adwaita. Then that should be fine, but it's probably mildly better to just have only a gtk-3.0 folder, to match Adwaita and make it a little bit simpler.

Reply:
@ids1024 I can get that simplified, but in the interest of allowing QA to better prioritize that, I think it would be better to get that set up in a separate PR once this is merged.